### PR TITLE
Splash screen Added

### PR DESCRIPTION
**Pull request by karanpatel2329**
#5 issue is fixed.
I have added a splash screen using flutter package -- flutter_native_splash.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
> #5 issue is fixed.
> I have added a splash screen using flutter package -- flutter_native_splash.

Please follow the guidelines given on slack. You are requested to provide a clip/snapshot of the changes for all the PRs

**Comment by karanpatel2329** [comment]
> > #5 issue is fixed.
> > I have added a splash screen using flutter package -- flutter_native_splash.
> 
> Please follow the guidelines given on slack. You are requested to provide a clip/snapshot of the changes for all the PRs

splash screen
homepage

**Review by anushbhatia — CHANGES_REQUESTED** [review]
https://www.flaticon.com/free-icon/nurse_1788280?term=nurse&related_id=1788280
You can use this logo and avoid using capital letters in the splash screen.

File: web/splash/style.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+body, html {
+      margin:0;
```
Comment (anushbhatia) [inline web/splash/style.css:2]: What is the use of modfying this we aren't using flutter web for this project

File: pubspec.yaml [hunk 2]
```diff
@@ -27,15 +27,15 @@ dependencies:
...
+  cupertino_icons:
   charts_flutter: ^0.9.0
   wakelock: ^0.1.4+1
   camera: 0.7.0+4
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-
+  flutter_native_splash:
```
Comment (anushbhatia) [inline pubspec.yaml:38]: Mention this above the dev_dependencies

**Review by anushbhatia — COMMENTED** [review]
Also, don't push .idea named files to add them in gitignore so that they won't be pushed

**Comment by smaranjitghose** [comment]
> > > #5 issue is fixed.
> > > I have added a splash screen using flutter package -- flutter_native_splash.
> > 
> > 
> > Please follow the guidelines given on slack. You are requested to provide a clip/snapshot of the changes for all the PRs
> 
> splash screen
> homepage

Could you upload a clip of the splash screen please from the emulator? If it's just a static image, that's not good enough

**Comment by anushbhatia** [comment]
Any update you can create a new pr with changes it will work now cause all other files are commented.

**Pull request closed without merging**